prusti-specs/src/lib.rs

```diff
@@ -211,7 +211,14 @@ fn generate_for_after_expiry_if(attr: TokenStream, item: &untyped::AnyFnItem) ->
 }
 
 /// Generate spec items and attributes to typecheck and later retrieve "pure" annotations.
-fn generate_for_pure(_attr: TokenStream, item: &untyped::AnyFnItem) -> GeneratedResult {
+fn generate_for_pure(attr: TokenStream, item: &untyped::AnyFnItem) -> GeneratedResult {
+    if !attr.is_empty() {
+        return Err(syn::Error::new(
+            attr.span(),
+            "The `#[pure]` attribute does not take parameters"
```

Comment:
Really minor comment: Rust's error messages usually start with a lowercase letter.

Reply:
adjusted; also fixed a copy/paste-mistake in the error message.